pxr/CMakeLists.txt

```diff
@@ -23,6 +23,13 @@ pxr_core_epilogue()
 
 export(PACKAGE pxr)
 
+if(TBB_DIR)
+    set(PXR_FIND_TBB_IN_CONFIG ON)
+endif()
+if(OpenSubdiv_DIR)
+    set(PXR_FIND_OPENSUBDIV_IN_CONFIG ON)
+endif()
```

Comment:
Personally, I would find this more readable if these variables were set in `Packages.cmake`, in the same branches where we first check to see if the config packages were found.  (And also explicitly set to OFF if not found.)

I can see the thinking of wanting to restrict Pixar-specific variables to the `pxr` subdirectory - but really, everything in the `cmake` subdir is already very Pixar-specific, and there's already examples of setting `PXR_XXX` variables in there... so I'd rather it easier to read.

Reply:
Thanks, changed in 08c8ca97d55e5204c8d92bb59d953df176183bda